static/css/components/work.less

```diff
@@ -40,7 +40,7 @@ div.editionAbout {
 
   p {
     margin: 1em 0;
-    font-size: 13px;
+    font-size: @font-size-body-medium;
```

Comment:
@0simoo are the changes in this file intentional? If so, can you make a comment on why they're needed?
If not, can you revert them?

Thanks!

Reply:
My mistake! I'm reverting them now